llm/qwen/README.md

```diff
@@ -0,0 +1,127 @@
+# Serving Qwen1.5 on Your Own Cloud
+
+[Qwen1.5](https://github.com/QwenLM/Qwen1.5) is one of the top open LLMs.
```

Comment:
Maybe add: As of Feb 2024, [Qwen1.5-72B-Chat](https://qwenlm.github.io/blog/qwen1.5/) is ranked higher than [Mixtral-8x7b-Instruct-v0.1](https://mistral.ai/news/mixtral-of-experts/) on the LMSYS [Chatbot Arena Leaderboard](https://chat.lmsys.org/).